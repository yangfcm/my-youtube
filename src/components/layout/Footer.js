import React from "react";

const Footer = () => {
  return (
    <div className="font-weight-light text-center bg-light py-3">
      My Youtube player developed by Fan Y.
    </div>
  );
};

export default Footer;
