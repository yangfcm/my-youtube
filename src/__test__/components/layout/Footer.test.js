import React from "react";
import { shallow } from "enzyme";
import Footer from "components/layout/Footer";

describe("Test Footer component", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Footer />);
  });

  it("should render component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
