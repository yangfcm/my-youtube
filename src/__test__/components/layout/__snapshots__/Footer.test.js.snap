// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Footer component should render component 1`] = `
<div
  className="font-weight-light text-center bg-light py-3"
>
  My Youtube player developed by Fan Y.
  <div
    className="font-weight-light text-center"
  >
    Powered by 
    <a
      href="https://reactjs.org"
      target="__blank"
    >
      React
    </a>
  </div>
</div>
`;
